src/d/ir/expression.d

```diff
@@ -542,6 +542,8 @@ enum CastKind {
 	Invalid,
 	IntToPtr,
 	PtrToInt,
+	IntegerToFloat,
+	FloatToInteger,
```

Comment:
For better or worse, FloatToInt or something like that seems more in line with existing entries in the enum.

Reply:
Good point. It was initially IntegralToFloat but then I saw whatever the fuck D actually does

Reply:
I changed it